addons/s3/utils.py

```diff
@@ -143,3 +149,35 @@ def get_bucket_prefixes(access_key, secret_key, prefix, bucket_name):
             )
 
     return folders
+
+
+def update_folder_names():
+    NodeSettings = apps.get_model('addons_s3', 'NodeSettings')
+
+    # Update folder_id for all records
+    NodeSettings.objects.update(folder_id=Concat(F('folder_id'), Value(':/')))
```

Comment:
This has hit staging already, and if we're deploying without downtime, prod data could hypothetically already have a "new"-style path or two. `.exclude(folder_[thing]__contains=':/')` would keep us from stepping on our toes in the forward.

Reply:
Follow-up to this, actually -- the tests could easily cover this case with a `S3NodeSettingsFactory(folder_name='Folder 4:/ (Location 4)', folder_id='folder4:/')` initial case